content/terraform/v1.13.x (beta)/docs/language/tests/index.mdx

```diff
@@ -174,12 +175,38 @@ Additionally, test assertions can directly reference outputs from current and pr
 
 ## Variables
 
-You can provide values for [Input Variables](/terraform/language/values/variables) within your configuration directly from your test files.
+You can both define variables for your test file and provide values for [Input Variables](/terraform/language/values/variables) within your configuration directly from your test files.
 
-The test file syntax supports `variables` blocks at both the root level and within `run` blocks. Terraform passes all variable values from the test file into all `run` blocks within the file. You can override variable values for a particular `run` block with values provided directly within that `run` block.
+The test file syntax supports `variable` blocks at the root level. These allow you to define new variable values that can be referenced and shared by providers, other variables, and run blocks defined within your test file.
 
 Adding to the test file from the [example](#example) above:
 
+```hcl
+# variables.tftest.hcl
+
+variable "test_input" {
+  type = string
+}
+
+run "uses_test_input" {
+
+  command = plan
+
+  variables {
+    bucket_prefix = var.test_input
+  }
+
+  assert {
+    condition     = aws_s3_bucket.bucket.bucket == var.test_input
+    error_message = "S3 bucket name did not match expected"
+  }
+}
+```
+
+In addition, the test file syntax supports `variables` blocks at both the root level and within `run` blocks. Terraform passes all variable values from the test file into all `run` blocks within the file. You can override variable values for a particular `run` block with values provided directly within that `run` block.
```

Comment:
```suggestion
In addition to defining variables at the root level, the test file syntax also supports optional `variables` blocks within `run` blocks. Terraform passes all variable values from the test file into `run` blocks within the same file. You can override variable values for a particular `run` block by defining values within the `run` block's `variables` block. 
```

Alternatively, we could get rid of "in addition" and just say "The test file syntax ... at both the root level and ..."

- Also, just confirming that variables in test files are scoped to that file, not the entire workspace, unlike terraform blocks that are scoped to the workspace.